velox/docs/functions/presto/map.rst

```diff
@@ -132,6 +132,14 @@ Map Functions
         SELECT map_top_n_keys(map(ARRAY['a', 'b', 'c'], ARRAY[1, 2, 3]), 2) --- ['c', 'b']
         SELECT map_top_n_keys(map(ARRAY['a', 'b', 'c'], ARRAY[1, 2, 3]), 0) --- []
 
+.. function:: map_top_n_values(map(K,V), n) -> array(V)
```

Comment:
do you plan to update this in this PR or in https://github.com/prestodb/presto/pull/24806?